packages/esm-patient-orders-app/src/order-basket/order-basket.workspace.tsx

```diff
@@ -122,6 +123,16 @@ const OrderBasket: React.FC<DefaultWorkspaceProps> = ({
               inline
```

Comment:
[InlineNotification](https://react.carbondesignsystem.com/?path=/docs/components-dropdown--overview) doesn't take an `inline` prop. We should remove this.

Reply:
Should remove this one too.